lldb/source/Target/Target.cpp

```diff
@@ -2747,6 +2747,15 @@ llvm::Optional<SwiftScratchContextReader> Target::GetSwiftScratchContext(
         log->Printf("not allowed to create a new context\n");
       return nullptr;
     }
+
+    // Call for its side effects of establishing the Swift scratch type system.
+    auto type_system_or_err =
+        GetScratchTypeSystemForLanguage(eLanguageTypeSwift, false);
+    if (!type_system_or_err) {
+      llvm::consumeError(type_system_or_err.takeError());
+      return nullptr;
```

Comment:
Did you add the if-error-return path?
IIUC, github doesn't let me see older variants of the PR...

Reply:
There's a Compare link each time a force push happens
 
<img width="856" alt="Screenshot 2023-06-07 at 6 21 51 PM" src="https://github.com/apple/llvm-project/assets/43429/f4118faa-22b9-420e-bcce-791955e9ed2a">

See https://github.com/apple/llvm-project/compare/62ac2706d6f10eda08a593adc62208c995868d7c..f5d4c14af8bc6a41e82f2b77ea2b052efcb17b19